tensorflow/docs_src/api_guides/cc/guide.md

```diff
@@ -1,4 +1,12 @@
 # C++ API
+
```

Comment:
Are changes to this doc intentional?

Reply:
Yes, this for instance was a doc fix merged directly into r1.4
https://github.com/tensorflow/tensorflow/pull/13858/files